recipes/lipo/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "lipo" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a943adcda8be0326774ac9a1fcca20b955f9746e58ef00586d145b018fd90ee4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<36]
```

Comment:
I think you can get rid of the `skip` and make this `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

and then just change `python` to `python >=3.6` under `host` and `run` below

Reply:
Right. Thanks for the pointer!